kubernetes/account-rest-service-deployment.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: account-rest-service-deployment
+  namespace: default
+  labels:
+    app: account-rest-service
+spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  replicas: 3
+  selector:
+    matchLabels:
+      app: account-rest-service
+  template:
+    metadata:
+      labels:
+        app: account-rest-service
+    spec:
+      containers:
+        - name: account-rest-service
+          image: 048246832408.dkr.ecr.ap-northeast-1.amazonaws.com/account-rest-service:release.${IMAGE_TAG}
```

```suggestion
          image: 048246832408.dkr.ecr.ap-northeast-1.amazonaws.com/account-rest-service:${IMAGE_TAG}
```

変数として使用されるならばこうするのが適切だと思います
環境変数の方に手を加えるのがいいとおもいます

---

5fd6e85
修正しました。